pyccel/codegen/printing/pycode.py

```diff
@@ -845,6 +845,13 @@ def _print_NumpyCountNonZero(self, expr):
 
         return "{}({})".format(name, arg)
 
+    def _print_ListAppend(self, expr):
+        method_name = expr.name
+        dotted_var = self._print(expr.args[0])
+        method_args = self._print(expr.args[1])
+
+        return f"{dotted_var}.{method_name}({method_args})\n"
```

Comment:
Are you sure that the `\n` is necessary? Have you checked your code to ensure that this is not being added after? Generally functions don't usually add `\n` themselves but I don't remember if functions which don't return results are an exception to this?

Reply:
yes i did check and somehow `\n` was not added after, in case of multiple `append()` calls i was getting this:
`a.append()a.append()`
ill check again tho after i fix the class

Reply:
Ok. No need to check again. I actually saw this code yesterday and `\n` is added automatically if the result is not assigned so it should be here for this function :+1: